Gemfile

```diff
@@ -33,22 +34,24 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+gem 'rubocop', require: false
```

Comment:
We usually lock the gem version to keep the dependencies under control, for example [this](https://github.com/wolox-training/pj-fa-rails/pull/4/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R50) gem

Reply:
Done